strawberry/directive.py

```diff
@@ -13,6 +17,18 @@ class DirectiveDefinition:
     locations: List[DirectiveLocation]
     description: Optional[str] = None
 
+    @property
+    def arguments(self) -> List[ArgumentDefinition]:
+        annotations = self.resolver.__annotations__
+        annotations = dict(islice(annotations.items(), 1, None))
```

Comment:
is this to remove `self`?

Reply:
Honestly, I'm not sure what it does. I just migrated it from here:
https://github.com/strawberry-graphql/strawberry/blob/9815872a78123cd80ae3fe2d9bf4e6b03262c402/strawberry/schema/types/directives.py#L13-L23

Reply:
ok! I think I’ll need to improve directives a bit in future :)